reviewers/JSCSChecker.js

```diff
@@ -0,0 +1,268 @@
+/**
+ * Copyright RIKSOF (Private) Limited 2016.
+ *
+ * JS Style Checker.
+ */
+
+checker = function () {
+  
+  /**
+   * Array of file paths.
+   */
+  this.checkedFiles = [];
+}
+
+/**
+ * Instance methods.
+ */
+checker.prototype = {
+  
+  SEVERITY_HIGH: 1,
+  SEVERITY_MEDIUM: 2,
+  SEVERITY_LOW: 3,
+  
+  /**
+   * Function is used to reset the checker for next pull review.
+   */
+  reset: function(  ) {
+    this.checkedFiles = [];
+  },
+  
+  /**
+   * Indicates to the caller if this checker is interested in given
+   * file.
+   *
+   * @param file   Relative path of file.
+   */
+  doesValidate: function( file ) {
+    var validates = false;
+    var included = [ '.js' ];
+    
+    included.forEach( function( e ) {
+      if ( file.substr( -e.length) === e ) {
+        validates = true;
+      }
+    });
+
+    return validates;
+  },
+  
+  /**
```

Comment:
Missing JSDoc @returns for function.
```javascript
  /**
```
Missing JSDoc parameter type for 'headSource'.
```javascript
  /**
```
Missing JSDoc parameter type for 'to'.
```javascript
  /**
```
Missing JSDoc parameter type for 'baseSource'.
```javascript
  /**
```
Missing JSDoc parameter type for 'from'.
```javascript
  /**
```